src/utils/types.d.ts

```diff
@@ -6,3 +6,9 @@ interface Balance {
   nonce: number | bigint;
   address: string;
 }
+
+interface Promise<T> {
```

Comment:
does anything use this new declaration?

Reply:
Yeah, this is a global declaration for all promises